homeassistant/components/roomba/strings.json

```
@@ -18,11 +18,11 @@
       },
       "link": {
         "title": "Retrieve Password",
-        "description": "Press and hold the Home button on {name} until the device generates a sound (about two seconds), then submit within 30 seconds."
+        "description": "Press and hold the Home button on {name} until the device generates a sound (about two seconds), then submit within 30 seconds. Please make sure that the iRobot app is not open on any device while you are going through this process"
```

Why not have it in a step-by-step way instead of a remark at the end? I can imagine lots of people (including myself) doing while reading, not first-read-then-do.

```suggestion
        "description": "Make sure that the iRobot app is not running on any device. Press and hold the Home button on {name} until the device generates a sound (about two seconds), then submit within 30 seconds."
```